library/Icinga/Util/CspHeader.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Icinga\Util;
+
+use Icinga\Web\Session;
+use Icinga\Web\Window;
+
+class CspHeader
```

Comment:
I would rename that class to just `Csp` and would like to use it as follows:

* `Csp::getStyleNonce()` returns the nonce for use in inline styles. We don't have `getScriptNonce()` yet but we might so it makes sense to think about separation right from the beginning.
* `Csp::createNonces()` creates the style nonce. May also create the script nonce in the future.
* `Csp::addHeader(Response)` adds the csp header to the response and also adds the window id header as we're working with `Window` internally.